cluster/images/provider-kubernetes-controller/Makefile

```diff
@@ -34,24 +34,28 @@ include ../../../build/makelib/image.mk
 
 img.build:
 ifneq (,$(BUILDX))
-	$(eval CONTAINER_CLI = docker buildx build --platform linux/$(ARCH))
+	$(eval CONTAINER_CLI = docker buildx build --platform linux/$(ARCH) $(BUILDX_ARGS))
+else
+ifneq (,$(shell which podman))
+	$(eval CONTAINER_CLI = podman build --arch $(ARCH) --format docker)
 else
 ifneq (,$(BUILDX_PLUGIN))
-	$(eval CONTAINER_CLI = $(BUILDX_PLUGIN) build --platform linux/$(ARCH))
+	$(eval CONTAINER_CLI = $(BUILDX_PLUGIN) build --platform linux/$(ARCH) $(BUILDX_ARGS))
 else
-	$(eval CONTAINER_CLI = docker)
+	$(eval CONTAINER_CLI = docker build)
+endif
 endif
 endif
 	@$(INFO) $(CONTAINER_CLI) $(IMAGE)
 	@cp Dockerfile $(IMAGE_TEMP_DIR) || $(FAIL)
 	@cp $(OUTPUT_DIR)/bin/$(OS)_$(ARCH)/provider $(IMAGE_TEMP_DIR) || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && $(SED_CMD) 's|BASEIMAGE|$(OSBASEIMAGE)|g' Dockerfile || $(FAIL)
-	@$(CONTAINER_CLI) $(BUILDX_ARGS) $(BUILD_ARGS) \
+	@$(CONTAINER_CLI) $(BUILD_ARGS) \
 		--build-arg ARCH=$(ARCH) \
 		--build-arg TINI_VERSION=$(TINI_VERSION) \
 		-t $(IMAGE) \
 		$(IMAGE_TEMP_DIR) || $(FAIL)
-	@$(CONTAINER_CLI) $(BUILDX_ARGS) $(BUILD_ARGS) \
+	@$(CONTAINER_CLI) $(BUILD_ARGS) \
```

Comment:
I think it would be better to build multi-arch image as `latest` (here https://github.com/IBM/crossplane-provider-kubernetes/blob/master/ibm/Makefile.common.mk in terget `images`)

Reply:
Good remark. This was actually required for GitHub actions to work properly but since we fixed building locally and configured prow, it could be possible to delete this change.